Automattic/jetpack
# Refactor My Jetpack's backup state management

Related https://github.com/Automattic/jetpack/pull/35697

## Proposed changes:
* Refactor My Jetpack's state management: backup endpoints

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:
* Activate the feature branch in the Jetpack Beta plugin using Jurassic Ninja
* Go to `/wp-admin/admin.php?page=my-jetpack`
* Check the "Network" tab in the inspector to make sure two queries are being executed; make sure there are no runtime errors in JS code
